lesson7-questions.txt

```diff
@@ -67,13 +90,20 @@ Try that last line.  You will see that it creates a subscription object, but tha
 object is not entirely empty.  Which attribute is set? Where does it get that
 value?
 
+new_subscription = user.subscriptions.new creates a new subcription, where the user_id attribute is automatically set to the id of the user instance that was used to create the new subscription.
 
 
 7. Suppose that user is an instance of User, and forum is an instance of Forum.  What
 Active Record Statement could you use to retrieve all the user's posts for that
 forum?  Hint: You need to have a where in your statement. Try this out in the console.
 You will see the SQL that is executed logged to the screen.  What is that SQL?
 
+Active Record Statement to retrieve all the specific user's posts for the specific
+forum is:
+  user_posts_in_forum = Post.where(user_id: user.id, forum_id: forum.id)
```

Comment:
this works but you could use the association directly with user.posts.where(forum: forum)